node-template/runtime/Cargo.toml

```
@@ -15,11 +15,10 @@ support = { package = "srml-support", path = "../../srml/support", default_featu
 primitives = { package = "substrate-primitives", path = "../../core/primitives", default_features = false }
 substrate-session = { path = "../../core/session", default-features = false }
 balances = { package = "srml-balances", path = "../../srml/balances", default_features = false }
-babe = { package = "srml-babe", path = "../../srml/babe", default-features = false }
-babe-primitives = { package = "substrate-consensus-babe-primitives", path = "../../core/consensus/babe/primitives", default-features = false }
+aura = { package = "srml-aura", path = "../../srml/aura", default_features = false }
+consensus-aura = { package = "substrate-consensus-aura-primitives", path = "../../core/consensus/aura/primitives", default_features = false }
```

```suggestion
aura-primitives = { package = "substrate-consensus-aura-primitives", path = "../../core/consensus/aura/primitives", default_features = false }
```

I don't know if it was named like this before but it makes more sense and neater.